packages/modelence/src/lock/helpers.ts

```diff
@@ -0,0 +1,180 @@
+import { randomBytes } from 'crypto';
+import { locksCollection } from './collection';
+
+/**
+ * Unique identifier for this application instance.
+ * Generated once per application instance to track which container owns which locks.
+ */
+const containerId = randomBytes(32).toString('base64url');
```

Comment:
Since this code is container specific, would it make more sense to call this whole directory of files `instances` or `containers` instead of locks?